lib/broccoli/ember-app.js

```diff
@@ -1397,11 +1401,21 @@ class EmberApp {
         throw new SilentError(`Style file cannot have the name of the application - ${this.name}`);
       }
 
+      let directory = '/app/styles';
+
+      if (experiments.MODULE_UNIFICATION) {
```

Comment:
This doesn't seem quite correct to me. We don't want to confuse the existing `this.trees.styles` with module unification at all. As written, when ember-cli itself supports module unification, we would no longer support app/styles, which seems bad...

What are you shooting for here?